test/e2e/network/ingress.go

```diff
@@ -1141,6 +1142,14 @@ var _ = SIGDescribe("Ingress API", func() {
 		framework.ExpectNoError(err)
 		framework.ExpectEqual(updatedStatus.Status.LoadBalancer, statusToUpdate.Status.LoadBalancer, fmt.Sprintf("updated object expected to have updated loadbalancer status %#v, got %#v", statusToUpdate.Status.LoadBalancer, updatedStatus.Status.LoadBalancer))
 
+		ginkgo.By("get /status")
```

Comment:
Nit: Comment here saying what this test is for

Reply:
Was following the same pattern that's already in the test 
- Line 1123 `ginkgo.By("patching /status")`
- Line 1136 `ginkgo.By("updating /status")`

Reply:
this looks fine to me as is